### ISSUE TITLE
pretrain LayoutLM v2 on new dataset

### ISSUE THREAD

**Issue by qifeishen** [microsoft/unilm#603]
Dear Authors, 
thanks for the great work.  Do you plan to release the pretraining part for LayoutLM v2?

**Comment by bonejay** [comment]
Yeah it would be a great help. Kind of tiresome that none of the document transformers (TILT, StructuralLM, layoutlm,StrucTexT) have release their pretraining code, although it is the most important factor to their performance.

**Comment by zzcgithub** [comment]
I hope so

**Comment by CheungZeeCn** [comment]
+1

**Comment by zzcgithub** [comment]
+2

**Comment by sudhirpol522** [comment]
+3

**Comment by rr191211** [comment]
I wonder when the source code will be released? Thanks

**Comment by ivsanro1** [comment]
> Yeah it would be a great help. Kind of tiresome that none of the document transformers (TILT, StructuralLM, layoutlm,StrucTexT) have release their pretraining code, although it is the most important factor to their performance.

Just a guess, but they do this on purpose to force you to use their pre-trained models, and also to have control over their models and implementation. They never release pretraining codes (also for MarkupLM)
